BadHabbit/BadHabbit/UserStories/SingUpScreen/SignUpViewController.swift

```diff
@@ -36,6 +36,17 @@ final class SignUpViewController: BaseViewController, SignUpViewProtocol {
 
         setupUI()
     }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+       navigationController?.setNavigationBarHidden(true, animated: animated)
```

Comment:
Doesn't design requires you to show navigationBar?

Reply:
on the rootviewController we don't have navigationBar